content/en/guide/v10/typescript.md

```diff
@@ -350,7 +358,7 @@ const Counter = ({ initial = 0 }) => {
 
 `useEffect` does extra checks so you only return cleanup functions.
```

Comment:
It seems this line is mere explanation about `useEffect`.  Since Hooks page is suitable, this line is unnecessary.